explorer/README.md

```diff
@@ -22,6 +22,7 @@ By default Scalar DL Explorer looks for the `client.properties` in the project d
 
 Scalar DL Explorer has four commands:
 
+- `explorer initialize`: used to register your certificate and contracts(GetContract, ScanContract). You have to execute this command one time only.
```

Comment:
```suggestion
- `explorer initialize`: used to register your certificate and contracts(GetContract, ScanContract). This command needs to be executed only one time.
```

Reply:
Thank you for review change.
Review changes updated in 135ba8e5358cdc9bfd02084406fd65ba68b3ddd4